home-application/pom.xml

```diff
@@ -78,13 +80,30 @@
     <build>
         <plugins>
             <plugin>
-                <groupId>org.springframework.boot</groupId>
-                <artifactId>spring-boot-maven-plugin</artifactId>
```

Comment:
Why did you remove this plugin? In order to run the app from jar we still need fat jar.

Reply:
that was not deliberately, will return plugin in the next commit